### DISCUSSION TITLE
Sensors grouping

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [HomeSpan/HomeSpan#177] [Q&A] [answered]
is it possible to somehow turn off sensor grouping? I want each sensor to display as a separate accessory.

at the moment I have 3 sensors that are in the "climate" group. or is it just happening on the ios side.

**Comment by HomeSpan** [comment]
That's all on the Home App (iOS) side.   However, the settings screen for the Accessory in the Home App may provide you with an option to "Show Accessories as Separate Tiles".

**Reply by aLAN-LDZ** [reply]
switches and lights are fine, problem is temperature and humidity.

I have two humidity sensors and one temperature sensor. It all comes together and only after clicking it can I see level 3

**Comment by HomeSpan** [comment]
Are you running them with a Bridge Service or simply as multiple Accessories?  Whichever one, may want to try to the opposite and see if that has an impact.

One of the problems with HomeKit is that the Home App tries to be too clever and there is no way to control how it renders Accessories.  Even worse is that this seems to change every few months with iOS update.  All of a sudden something that was represented one way in the Home App changes to something completely different.

**Comment by HomeSpan** [comment]
Some people use the EVE Elgato app.  It's free from the App Store and does a nice job rendering Home Kit Accessories.  There are also some other fee-based Home App alternatives, but I have not tried them.

**Comment by HomeSpan** [comment, marked as answer]
I had a chance to test various configurations (Bridge, No-Bridge, Extra Name Characteristics, etc.) but it seems that the Home App is keen on always grouping sensors together in one "circle" at the top.  As you noted, you can see all the individual sensors if you click the group, but there is unfortunately no option in the Home App for splitting them into their own distinct tiles.

I believe the "group circle" concept was added in iOS 14 (and persists in iOS 15), though in iOS 13 sensors were automatically shown as individual tiles.

I also verified that the EVE Elgato app does display each sensor as a separate item, so it's not a HomeSpan issue, but rather a "feature" of the Home App.
